### ISSUE TITLE
Repeated minor version

### ISSUE THREAD

**Issue by dimaqq** [hugovk/pypistats#66]
```
> pypistats python_minor requests
| category | percent |  downloads  |
|----------|--------:|------------:|
|      2.7 |  53.63% | 145,754,775 |
|      3.6 |  23.26% |  63,207,276 |
|      3.7 |  10.27% |  27,918,514 |
|      3.5 |   7.79% |  21,173,697 |
|      3.4 |   3.48% |   9,458,366 |
|      2.6 |   0.80% |   2,170,184 |
| null     |   0.73% |   1,978,215 |
|      3.8 |   0.04% |      97,944 |
|      3.3 |   0.01% |      34,798 |
|      3.2 |   0.00% |       3,278 |
|      3.9 |   0.00% |       2,764 |
|      2.8 |   0.00% |         240 |
|      3.1 |   0.00% |          16 |
|      2.5 |   0.00% |           4 |
|      3.1 |   0.00% |           1 |
| Total    |         | 271,800,072 |
```

How come is version `3.1` present twice?

**Comment by hugovk** [comment]
Thanks for the report, this looks like a bug.

Running verbose to check the data source URL:
```console
$ pypistats python_minor requests --verbose
API URL: https://pypistats.org/api/packages/requests/python_minor
Cache file: /Users/hugo/Library/Caches/pypistats/2019-07-30-https-pypistats-org-api-packages-requests-python-minor.json
Cache file exists
| category | percent |  downloads  |
|----------|--------:|------------:|
|      2.7 |  53.63% | 145,754,775 |
|      3.6 |  23.26% |  63,207,276 |
|      3.7 |  10.27% |  27,918,514 |
|      3.5 |   7.79% |  21,173,697 |
|      3.4 |   3.48% |   9,458,366 |
|      2.6 |   0.80% |   2,170,184 |
| null     |   0.73% |   1,978,215 |
|      3.8 |   0.04% |      97,944 |
|      3.3 |   0.01% |      34,798 |
|      3.2 |   0.00% |       3,278 |
|      3.9 |   0.00% |       2,764 |
|      2.8 |   0.00% |         240 |
|      3.1 |   0.00% |          16 |
|      2.5 |   0.00% |           4 |
|      3.1 |   0.00% |           1 |
| Total    |         | 271,800,072 |
```

Looking at https://pypistats.org/api/packages/requests/python_minor, there's a "3.10" which is probably being interpreted as an integer instead of a string when being displayed:


```json
...
{
"category": "3.1",
"date": "2019-07-18",
"downloads": 1
},
{
"category": "3.1",
"date": "2019-07-23",
"downloads": 1
},
{
"category": "3.10",
"date": "2019-07-18",
"downloads": 1
},
...
```

**Comment by hugovk** [comment]
Thanks again for the report, this has been fixed and released in pypistats 0.7.1.

By the way, Python 3.10 hasn't been released, but https://github.com/asottile/python3.10 is a test build to explicitly look for bugs with a 3.10, two-digit minor version number. It found one here!
